src/scripts/main.js

```diff
@@ -1,3 +1,44 @@
 'use strict';
 
-// write code here
+const tabBody = document.querySelector('tbody');
+
+const personsInfo = tabBody.innerText.split('\n');
+
+const persons = personsInfo.map(el => el.split('\t'));
+
+persons.pop();
+
+const personsInTrueFormat = persons.map(el => (
+  {
+    Name: `${el[0]}`,
+    Position: `${el[1]}`,
+    Age: `${el[2]}`,
+    Salary: `${el[3]}`,
+  })
+);
+
+let selected = '';
+
+function byField(field) {
+  return (a, b) => a[field] > b[field] ? 1 : -1;
+};
+
+document.body.addEventListener('click', (event1) => {
```

Comment:
find table head and add event listener to it

```suggestion
thead.addEventListener('click', (event1) => {
```